src/test/java/edu/iis/powp/gui/TestPlotSoftPatterns.java

```diff
@@ -31,11 +31,13 @@ private static void setupPresetTests(Context context)
 		SelectTestFigureOptionListener selectTestFigureOptionListener2 = new SelectTestFigureOptionListener(2);
 		SelectComplexCommandOptionListener selectComplexCommandOptionListener1 = new SelectComplexCommandOptionListener(1);
 		SelectComplexCommandOptionListener selectComplexCommandOptionListener2 = new SelectComplexCommandOptionListener(2);
+		SelectComplexCommandOptionListener selectComplexCommandOptionListener3 = new SelectComplexCommandOptionListener(3);
```

Comment:
Type of the test based on int parameter makes code difficult to understand (enum would be better).